website/docs/tutorials/guides/testing.md

```diff
@@ -94,29 +116,42 @@ namespace StorageContract:
 
     func get_balance() -> (res : felt):
     end
+    
+    func get_id() -> (res: felt):
+    end
 end
 
 @external
 func test_proxy_contract{syscall_ptr : felt*, range_check_ptr}():
     alloc_locals
 
-    local contract_address : felt
-    %{
-        # We deploy contract and put its address into a local variable
-        ids.contract_address = deploy_contract("./src/storage_contract.cairo").contract_address 
-    %}
+    local contract_a_address : felt
+    # We deploy contract and put its address into a local variable. Second argument is calldata array
+    %{ ids.contract_a_address = deploy_contract("./src/commands/test/examples/basic_with_constructor.cairo", [100, 0, 1]).contract_address %}
 
+    let (res) = StorageContract.get_balance(contract_address=contract_a_address)
+    assert res.low = 100
+    assert res.high = 0
+
+    let (id) = StorageContract.get_id(contract_address=contract_a_address)
+    assert id = 1
+  
     StorageContract.increase_balance(
         contract_address=contract_address,
-        amount=5
+        amount=Uint256(50, 0)
     )
 
     let (res) = StorageContract.get_balance(contract_address=contract_address)
-    assert res = 5
+    assert res.low = 150
+    assert res.high = 0
     return ()
 end
 ```
 
+:::info
+Please refer to https://www.cairo-lang.org/docs/hello_starknet/more_features.html#passing-tuples-and-structs-in-calldata on how to serialize your constructor arguments to array of integers
+:::
+
```

Comment:
Can you replace "Representing different data structures in `ret_data`" section with the info box?